client/package.json

```diff
@@ -36,5 +36,11 @@
     "vite": "^4.5.5",
     "vue-eslint-parser": "^9.4.2",
     "vue-tsc": "^1.8.5"
+  },
+  "packageManager": "yarn@4.5.1+sha256.dc2e45a77f87a9c2a76708d098ab54ca0cfdd5ec2cf7d9e3c1f24bf786a2383c",
```

Comment:
> (do you need to specify yarn version berry to run tox -e binary or just tox?).

Locally I am able to run `tox` and `tox -e binary` in a terminal with `yarn v1.22.0` and still have it build correctly so long as I am using `node 20+` with `corepack` enabled. I believe this line is why.

Reply:
After some investigation, yes this line combined with `corepack` is the magic. Once it is enabled node is able to detect the correct yarn version. I have updated the ci yaml to reflect this.

Corepack is also what allows us to switch to different projects using different versions without manually setting a new version each time. Huzzah!

https://nodejs.org/api/corepack.html#configuring-a-package